README.md

```diff
@@ -2,168 +2,136 @@
+
+Next, install Nginx to use it as the web server:
 
 ```bash
 sudo apt install nginx
 ```
-### 3. Created an alternative  HTML File
-A simple HTML file was created to be served by Nginx. 
 
-The following directory structure and HTML file were created:
+### 3. Create an alternative  HTML File
+
+Create a simple HTML file to be served by Nginx.
+
+Create the following directory structure and HTML file:
 
 ```bash
 sudo mkdir /var/www/alternatives
 sudo touch /var/www/alternatives/alternative-index.html
 ```
-The HTML file was edited with the following content:
+
+Edit the HTML file with the following content:
 
 ```bash
 sudo nano /var/www/alternatives/alternative-index.html
-
 ```
+
 The content was as follows:
 
-```html 
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Hello Nginx</title>
-  <style>
-    body {
-      font-family: Arial, sans-serif;
-      background: #f4f4f4;
-      text-align: center;
-      padding: 100px;
-    }
-    h1 {
-      color: #2c3e50;
-    }
-    p {
-      font-size: 1.2em;
-      color: #555;
-    }
-  </style>
-</head>
-<body>
-  <h1>Hello Nginx</h1>
-  <p>I just set up my server on the cloud and I am ready for the next pro>
-</body>
-</html>
-```
-The file was saved and the editor was exited.
+[index.html](Docs/index.html)
 
-### 4. Configured Nginx Server Block
+Save the file and exit the editor.
 
-A new Nginx configuration file was created for the "alternatives" website. This specified the server's root directory and the default index file.
+### 4. Configure Nginx Server Block
 
-The following configuration was added:
+Create a new Nginx configuration file for the website (e.g., “alternatives”). Specify the server’s root directory and the default index file.
+
+Open the configuration file with:
 
 ```bash
 sudo nano /etc/nginx/sites-enabled/alternatives
 ```
 
-The following server block was added:
-
-```bash
-server {
-    listen 8081;
-    listen [::]:8081;
-
-    root /var/www/alternatives;
-    index alternative-index.html;
+Add the following server block:
 
-    location / {
-        try_files $uri $uri/ =404;
-    }
-}
-```
-The file was saved and the editor was exited.
+[Nginx_Config](Docs/Nginx_Config.txt)
 
+Save the file and exit the editor.
 
-### 5. Restarted Nginx
+### 5. Restart Nginx
 
-After updating the configuration, Nginx was restarted to apply the changes:
+Restart Nginx to apply the configuration changes:
 
 ```bash
 sudo systemctl restart nginx.service
 ```
 
-
 ## Github configuration 
 
-### 1. Configured Git Globally
+### 1. Configure Git Globally
 
-To ensure that commits made on the server were associated with the GitHub account, Git was configured with the user name and email.
+Set Git user name and email globally on the server to associate commits with the GitHub account:
 
-The following commands were run to set the GitHub credentials globally:
 ```bash
 git config --global user.email "e1pmiS@github.com"
 git config --global user.name "e1pmiS"
```

Comment:
This should be your E-Mail Address and Name, e.g. for me it would be `sven@mail.com` and `Sven Meier`